plans/video-generation.md

```
@@ -0,0 +1,1140 @@
+    # ... existing fields ...
+    
+    # Secondary model config for auxiliary tasks (video intent analysis)
+    secondary_model_config: Optional[dict] = field(default_factory=dict)
+```
+
+---
+
+## 7. 涉及文件清单
+
+### 新增文件
+| 文件路径 | 说明 |
```

markdownlint-cli2 (0.21.0)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)